### PR TITLE
feat(route): Added TheVerge Hubs

### PR DESCRIPTION
**Pull request by vbali**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/theverge
/theverge/android
/theverge/apple
/theverge/apps
/theverge/blackberry
/theverge/culture
/theverge/gaming
/theverge/hd
/theverge/microsoft
/theverge/mobile
/theverge/photography
/theverge/policy
/theverge/web
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [ ] 新的路由 New Route
  - [ ] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [ ] 文档说明 Documentation
  - [ ] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/theverge/router.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module.exports = (router) => {
-    router.get('/', require('./index'));
+    router.get('/:hub?', require('./index'));
```
Comment (TonyRL) [inline lib/v2/theverge/router.js:2]: Please update the docs and maintainer.js as well

File: lib/v2/theverge/index.js [hunk 2]
```diff
@@ -3,7 +3,15 @@ const parser = require('@/utils/rss-parser');
 const cheerio = require('cheerio');
 
 module.exports = async (ctx) => {
-    const feed = await parser.parseURL('https://www.theverge.com/rss/index.xml');
+    let link;
+
+    if (ctx.params.hub) {
+        link = `https://www.theverge.com/${ctx.params.hub}/rss/index.xml`;
+    } else {
+        link = `https://www.theverge.com/rss/index.xml`;
```
Comment (TonyRL) [inline lib/v2/theverge/index.js:11]:
```suggestion
        link = 'https://www.theverge.com/rss/index.xml';
```

File: lib/v2/theverge/router.js (hunk 1 above)
Comment (vbali) [reply lib/v2/theverge/router.js:2]: Done, updated.

File: lib/v2/theverge/index.js (hunk 2 above)
Comment (vbali) [reply lib/v2/theverge/index.js]: Fixed the string quotes.

File: docs/en/new-media.md [hunk 3]
```diff
@@ -764,7 +764,22 @@ Compared to the official one, this feed:
 
 ### The Verge
 
-<RouteEn author="HenryQW" example="/theverge" path="/theverge">
+<RouteEn author="HenryQW" example="/theverge" path="/theverge/:hub? :paramsDesc="['Hub，see below, All Posts by default']">
```
Comment (TonyRL) [inline docs/en/new-media.md:767]:
```suggestion
<RouteEn author="HenryQW vbali" example="/theverge" path="/theverge/:hub? :paramsDesc="['Hub, see below, All Posts by default']">
```

File: docs/new-media.md [hunk 4]
```diff
@@ -1359,7 +1359,22 @@ IPFS 网关有可能失效，那时候换成其他网关。
 
 ### The Verge
 
-<Route author="HenryQW" example="/theverge" path="/theverge">
+<RouteEn author="HenryQW" example="/theverge" path="/theverge/:hub? :paramsDesc="['枢纽，见下表，默认为 All Posts']">
```
Comment (TonyRL) [inline docs/new-media.md:1362]:
```suggestion
<RouteEn author="HenryQW vbali" example="/theverge" path="/theverge/:hub? :paramsDesc="['枢纽，见下表，默认为 All Posts']">
```

File: lib/v2/theverge/maintainer.js [hunk 5]
```diff
@@ -1,3 +1,4 @@
 module.exports = {
     '/': ['HenryQW'],
+    '/:hub?': ['vbali'],
```
Comment (TonyRL) [inline lib/v2/theverge/maintainer.js:3]:
```suggestion
    '/:hub?': ['HenryQW', 'vbali'],
```

**Pull request merged**